package.json

```diff
@@ -9,6 +9,7 @@
     "@fortawesome/fontawesome-svg-core": "^1.2.8",
     "@fortawesome/free-solid-svg-icons": "^5.5.0",
     "@fortawesome/react-fontawesome": "^0.1.3",
+    "@svgr/webpack": "^4.3.3",
```

Comment:
Also not necessary. Also, please refrain from adding packages without asking first. I prefer to keep the package list small.

Reply:
Makes sense, hence removing the same